src/NPlug/Interop/LibVst.generated.cs

```diff
@@ -4036,7 +4036,7 @@ private static int canResize_Wrapper(IPlugView* self)
         /// On live resize this is called to check if the view can be resized to the given rect, if not
         /// adjust the rect to the allowed size.
         /// </summary>
-        private static partial ComResult checkSizeConstraint_ToManaged(IPlugView* self, LibVst.ViewRect* rect);
+        private static partial ComResult checkSizeConstraint_ToManaged(IPlugView* self,ref LibVst.ViewRect* rect);
```

Comment:
Same here and below, no need to change the native part. It is already correct and expect a pointer to ViewRect. Remove the ref.